### PR TITLE
Inter channel

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: conda_concourse_ci/uploads.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ def _base_task(upload_job_name):
             }}
 
 
+def upload_staging_channel(user, package_path):
+    """
+    Upload to anaconda.org using user.
+    """
+    cmd = 'upload --skip-existing --force -u {} {}'.format(user, package_path)
```
Comment (jjhelmus) [inline conda_concourse_ci/uploads.py:39]: Is a token required here?

**Comment by jjhelmus** [comment]
There is still one test failing and a few flake8 errors:
```
_____________________________ test_submit_one_off ______________________________
Traceback (most recent call last):
  File "/home/travis/build/conda/conda-concourse-ci/tests/test_execute.py", line 131, in test_submit_one_off
    config_root_dir=test_config_dir)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/execute.py", line 1218, in submit_one_off
    pass_throughs=pass_throughs, **kwargs)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/execute.py", line 1041, in compute_builds
    pass_throughs=pass_throughs, skip_existing=skip_existing)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/execute.py", line 117, in collect_tasks
    config=config)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/compute_build_graph.py", line 425, in construct_graph
    recipes_dir, config=config, finalize=finalize)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/compute_build_graph.py", line 216, in add_recipe_to_graph
    recipes_dir=recipes_dir, finalize=finalize)
  File "/home/travis/build/conda/conda-concourse-ci/conda_concourse_ci/compute_build_graph.py", line 478, in add_dependency_nodes_and_edges
    metadata = graph.nodes[node]['meta']
TypeError: 'method' object is not subscriptable
----------------------------- Captured stdout call -----------------------------

**Comment by katietz** [comment]
No, there is no token required AFAIK. See also the comment below in the
other 'anaconda upload' method. It explicit claims, either user or token.

Regards
Kai

**Comment by katietz** [comment]
The issue about 'TypeError: 'method' object is not subscriptable ' is the
use of networkx <2. Therefore I added to conda.recipe the requirement to
use >=2.0 version.
That issue is well described on migration advices from 1.9 to 2.x networkx

Hope that fixes it. You might run into pip issues, and you might need to
update pip package?

**Comment by jjhelmus** [comment]
LGTM, thanks @katietz

**Pull request merged**